src/encord_active/analysis/metrics/image/sharpness.py

```diff
@@ -0,0 +1,47 @@
+import math
+from typing import Optional, cast
+
+import torch
+
+from encord_active.analysis.metric import (
+    ImageObjectOnlyOutputBatch,
+    MetricDependencies,
+    ObjectOnlyBatchInput,
+    OneImageMetric,
+)
+from encord_active.analysis.types import (
+    BoundingBoxTensor,
+    ImageBatchTensor,
+    ImageTensor,
+    MaskTensor,
+    MetricBatchDependencies,
+    MetricResult,
+)
+from encord_active.analysis.util import laplacian2d
+from encord_active.db.metrics import MetricType
+
+
+class SharpnessMetric(OneImageMetric):
+    def __init__(self) -> None:
+        super().__init__(
+            ident="metric_sharpness",
+            long_name="Sharpness",
+            desc="",
+            metric_type=MetricType.NORMAL,
+        )
+
+    def calculate(
+        self, deps: MetricDependencies, image: ImageTensor, mask: Optional[MaskTensor], bb: Optional[BoundingBoxTensor]
+    ) -> MetricResult:
+        # Max range of laplacian kernel = [-4, 4] * 255 = [-1020, 1020]
+        # Max value of variance = (2.0 * 1020.0 * 1020.0) = 2080800.
```

Comment:
Where does this multiplication with 2 come from, I think It should be 1020*1020=1040400

Reply:
```
>>> torch.var(4 * torch.tensor([-255.0,255.0]))
tensor(2080800.)
```